pages/dug_10/4-osfv-status.md

```diff
@@ -0,0 +1,374 @@
+
+Manual tests will be slowly transferred to OSFV to reduce redundancy and improve coherence
+
+<center><img src="/img/dug_10/osfv-status/manual-test-1.png" width=500/></center>
+<center><img src="/img/dug_10/osfv-status/manual-test-2.png" width=500/></center>
+
+---
+
+# <center> Adding HW tests to OSFV CI </center>
+
+</br>
+Performing CI tests on real hardware will greatly improve the reliability of OSFV
+
+- The work is in progress
+- Two representatives prepared as a start
+    <center><table>
+    <tr>
+    <td>MSI Z690 DDR4 i5 14600k</td><td>PC Engines APU3C AMD GX-412TC</td>
+    </tr>
+    <tr>
+    <td><img src="/img/dug_10/osfv-status/z690.jpeg" width=300/></td>
+    <td><img src="/img/dug_10/osfv-status/apu3c.png" width=300/></td>
+    </tr>
+    </table></center>
+
+---
+
+# <center> Adding HW test to CI </center>
+## <center> Automatic test scope determining </center>
+
+<center><img src="/img/dug_10/osfv-status/auto-ci-scope.png" width=470/></center>
+<!--
+https://excalidraw.com/#json=qmgEKuh9t2q5LzVA6gHZ7,DBQpMBlSFCkvUrsJ3lDkoQ
+-->
+---
+
+# <center> Power On finally reliable </center>
+
+OSFV CLI uses the Power LED to determine the state of the platform
+```shell
+fgolas in ~ λ osfv_cli rte --rte_ip 192.168.10.198 pwr pwr_led
+DUT model retrieved from snipeit: V1410
+Using rte command is invasive action, checking first if the device is not used...
+Asset 150 is already checked out by you
+Power LED state: OFF
+
+fgolas in ~ λ osfv_cli rte --rte_ip 192.168.10.198 pwr on
+DUT model retrieved from snipeit: V1410
+Using rte command is invasive action, checking first if the device is not used...
+Asset 150 is already checked out by you
+Powering on...
+
+fgolas in ~ λ osfv_cli rte --rte_ip 192.168.10.198 pwr pwr_led
+DUT model retrieved from snipeit: V1410
+Using rte command is invasive action, checking first if the device is not used...
+Asset 150 is already checked out by you
+Power LED state: ON
+```
+
+---
+
+# <center> Power On finally reliable </center>
+
+```shell
+fgolas in ~ λ osfv_cli rte --rte_ip 192.168.10.198 pwr pwr_led
+DUT model retrieved from snipeit: V1410
+Using rte command is invasive action, checking first if the device is not used...
+Asset 150 is already checked out by you
+Power LED state: ON
+
+fgolas in ~ λ osfv_cli rte --rte_ip 192.168.10.198 pwr off
+DUT model retrieved from snipeit: V1410
+Using rte command is invasive action, checking first if the device is not used...
+Asset 150 is already checked out by you
+Powering off...
+
+fgolas in ~ λ osfv_cli rte --rte_ip 192.168.10.198 pwr pwr_led
+DUT model retrieved from snipeit: V1410
+Using rte command is invasive action, checking first if the device is not used...
+Asset 150 is already checked out by you
+Power LED state: OFF
+```
+
+---
+
+# <center> Priorities for the future </center>
+
+* Full SeaBIOS support is still on the roadmap
+* CI tests on real hardware on every PR are being worked on
```

Comment:
Let's say it's the main priority for the next quarter and put it on top.

In general, we want to ensure greater stability of the environment and make sure that introducing one change for one suite/platform does not break the other. This is a very complex environment with a lot of moving parts, sometimes with different quirks needed to be applied to different hardware platforms to automate testing on them.

Reply:
And the top goal we have is shortening the release cycle. We have identified that this is one of the major factors (fixing configs and tests with each regression cycle), and we want to minimize it.